rasa/core/policies/ensemble.py

```diff
@@ -69,6 +70,21 @@ def __init__(
         self._check_priorities()
         self._check_for_important_policies()
 
+        self._rule_only_slots, self._rule_only_loops = self._get_rule_only_slots_loops()
+
+    def _get_rule_only_slots_loops(self) -> Tuple[List[Text], List[Text]]:
+        rule_policy = next(
+            (policy for policy in self.policies if isinstance(policy, RulePolicy)),
+            None,
+        )
+        rule_only_slots = (
+            rule_policy.lookup.get(RULE_ONLY_SLOTS, []) if rule_policy else []
+        )
+        rule_only_loops = (
+            rule_policy.lookup.get(RULE_ONLY_LOOPS, []) if rule_policy else []
+        )
```

Comment:
can we make a public method on `RulePolicy` to get these (e.g. `get_slots_used_only_in_rules`)? Directly accessing these seems a bit fragile to me.